TheFourFifths/consus-client
# TFF-92: Item is already checked out

### Summary

Displays an error message when trying to complete a checkout with a cart containing unavailable items.

**NOTE: Review and merge this first: https://github.com/TheFourFifths/consus/pull/19**

### Checklist


Finally, is it ready to be reviewed and merged: yes :christmas_tree: 

### How to Review

1. Check out `consus:alternate-checkout-TFF-75-story` with git.
2. Build and start the server.
3. Check out `consus-client:item-unavailable-TFF-92` with git.
4. Build the client.
5. Start two clients, "A" and "B".
6. Enter student id `123456` into both A and B.
7. Enter item id `iGwEZUvfA` into both A and B.
8. Complete checkout on A.
9. Try to complete checkout on B.
10. Observe the appropriate error message on B: `This item is already checked out by another student.`

### Links

- [TFF-92](https://msoese.atlassian.net/browse/TFF-92)
- [Sister PR (required)](https://github.com/TheFourFifths/consus/pull/19)
